overrides/ios-override.json

```diff
@@ -38,17 +38,40 @@
                 }
             }
         },
+        "messageBridge": {
+            "exceptions": [],
+            "settings": {
+                "aiChat": "enabled",
```

Comment:
this should be disabled by default, the `domains` part will override it.

Reply:
done